vllm_ascend/envs.py

```diff
@@ -36,6 +36,8 @@
     lambda: bool(int(os.getenv("COMPILE_CUSTOM_KERNELS", "1"))),
     "VLLM_ENABLE_MC2":
     lambda: bool(int(os.getenv("VLLM_ENABLE_MC2", '0'))),
+    "VLLM_ENABLE_CV_PARALLEL":
```

Comment:
use additional_config instead of env, since this change is only used for torchair GE mode. like https://github.com/vllm-project/vllm-ascend/pull/839 does, there are 3 new config option. 
how about
```
{
   "additional_config": {
      "torchair_graph_config": {
         "enable": True,
         "enable_cv_parallet": True,
         "batch_sizes": "12345",
         "batch_sizes_init": True
      } 
  }
}

cc @zzzzwwjj 
```